### DISCUSSION TITLE
Is there a way to upload an ogg file?

### DISCUSSION THREAD

**Discussion by elyk17** [abramenal/cypress-file-upload#295] [Q&A]
My company works with .ogg files for hold music. Is there a way to setup the code to allow additional file extensions such as mp3 or ogg?

**Comment by abramenal** [comment]
Hi there!
I am sure this is possible, though I did not try it myself.
What you can do is check out this issue related to video upload - it might be that same code works with off.
If you can play around and share results, I am happy to create a documentation later on.
